crcmocks/templates/user_list.html

```diff
@@ -20,16 +20,20 @@
             <th>email</th>
             <th>org_id</th>
             <th>account_number</th>
+            <th>permissions</th>
+            <th>entitlements</th>
         </tr>
         {% for ruser in rusers %}
             <tr>
                 <td>{{ ruser.id }}</td>
```

Comment:
I changed this to `TinyDB` data rather than the `get_realm_users`, so that permissions/entitlements could be shown.